tox.ini

```diff
@@ -43,7 +43,7 @@ deps =
 
 commands =
     flake8 aiohttp examples tests
-    python setup.py check -rm
+    python3 setup.py check -rm
```

Comment:
Again we are in python3 virtual environment here.
Don't need `python3` command.